aries_cloudagent/protocols/endorse_transaction/v1_0/manager.py

```diff
@@ -797,7 +807,14 @@ async def endorsed_txn_post_processing(
             meta_data["context"]["public_did"] = public_did
 
             # Notify schema ledger write event
-            await notify_schema_event(self._profile, schema_id, meta_data)
+            if self._profile.settings.get("wallet.type") == "askar-anoncreds":
+                event_bus = self.profile.inject(EventBus)
+                await event_bus.notify(
+                    self._profile,
+                    SchemaRegistrationFinishedEvent.with_payload(meta_data),
+                )
+            else:
+                await notify_schema_event(self._profile, schema_id, meta_data)
```

Comment:
What do you think of just calling `AnonCredsIssuer.finish_schema` directly?

Reply:
I think that's a good idea. I was just following the existing pattern here but that would be simpler.